src/ec/packages/ec-component-accordion/accordion.story.js

```diff
@@ -1,36 +1,52 @@
 /* eslint-disable no-param-reassign */
 import { storiesOf } from '@storybook/html';
-import { withKnobs, text } from '@storybook/addon-knobs';
+import { withKnobs, text, select } from '@storybook/addon-knobs';
 import { withNotes } from '@ecl-twig/storybook-addon-notes';
+import { getExtraKnobs, buttonLabels } from '@ecl-twig/story-utils';
 import withCode from '@ecl-twig/storybook-addon-code';
-
 import defaultSprite from '@ecl/ec-resources-icons/dist/sprites/icons.svg';
 import demoData from './demo/data';
 
 import accordion from './ecl-accordion.html.twig';
 import notes from './README.md';
 
+const preprareAccordion = data => {
+  data.items.forEach((item, index) => {
+    const { id, level, toggle, content } = item;
+    const levels = [1, 2, 3, 4, 5, 6];
+    item.id = select(`item[${index}].id`, [id], id, buttonLabels.required);
+    item.level = select(
+      `item[${index}].level`,
+      levels,
+      level,
+      buttonLabels.required
+    );
+    item.toggle.label = text(
+      `items[${index}].toggle.label`,
+      toggle.label,
+      buttonLabels.required
+    );
+    item.content = text(
```

Comment:
item.content before item.toggle like accordion2

Reply:
like in accordion the Content should not come before the label